crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -536,35 +549,212 @@ pub enum AuthorizedotnetVoidStatus {
 impl From<AuthorizedotnetVoidStatus> for enums::AttemptStatus {
     fn from(item: AuthorizedotnetVoidStatus) -> Self {
         match item {
-            AuthorizedotnetVoidStatus::Approved => Self::VoidInitiated,
-            AuthorizedotnetVoidStatus::Declined | AuthorizedotnetVoidStatus::Error => Self::Failure,
+            AuthorizedotnetVoidStatus::Approved => Self::Voided,
+            AuthorizedotnetVoidStatus::Declined | AuthorizedotnetVoidStatus::Error => {
+                Self::VoidFailed
+            }
```

Comment:
HeldForReview should be mapped to VoidInitiated

Reply:
Done